test/recurrent/rnn-time-step.js

```diff
@@ -0,0 +1,134 @@
+import assert from 'assert';
+import RnnTimeStep from '../../src/recurrent/rnn-time-step';
+import LSTMTimeStep from "../../src/recurrent/lstm-time-step";
+
+/* NOTE: TimeStep here is deprecated though being committed as something new, it is the first feature we want using
+ recurrent.js because it is simple one of the simplest recurrent neural networks and serves as a baseline to completing
+ the GPU architecture.   This test is written so as to create the baseline we can measure against.
+ We get this working, we have a baseline, we finish recurrent.js, we change the world.
```

Comment:
You are too optimistic here, can you word it a bit more gloomily? 
![tenor](https://user-images.githubusercontent.com/2294139/38970987-f466b526-4355-11e8-9a37-724945c288af.gif)

Reply:
Could your give me an example of what you are looking for?